### ISSUE TITLE
do you plan to release IC-Light SDXL?

### ISSUE THREAD

**Issue by mikebilly** [lllyasviel/IC-Light#79]

**Comment by SoonNOON** [comment]
import torch
from diffusers import StableDiffusionXLPipeline, DPMSolverMultistepScheduler

# Function to create the pipeline
def create_pipeline(pipeline_type='text-to-image', scheduler=None, model_name=None):
    """Creates and returns a Stable Diffusion pipeline based on the specified type."""
    
    if scheduler is None:
        scheduler = DPMSolverMultistepScheduler(
            num_train_timesteps=1000,
            beta_start=0.00085,
            beta_end=0.012,
            algorithm_type="sde-dpmsolver++",
            use_karras_sigmas=True,
            steps_offset=1
        )
    
    if model_name == 'sdxl':
        return StableDiffusionXLPipeline.from_pretrained('stablediffusionapi/sdxl-1.0', torch_dtype=torch.float16)
    
    common_kwargs = {
        'vae': vae,
        'text_encoder': text_encoder,
        'tokenizer': tokenizer,
        'unet': unet,
        'scheduler': scheduler,
        'safety_checker': None,
        'requires_safety_checker': False,
        'feature_extractor': None,
        'image_encoder': None
    }
    
    if pipeline_type == 'text-to-image':
        return StableDiffusionPipeline(**common_kwargs)
    elif pipeline_type == 'image-to-image':
        return StableDiffusionImg2ImgPipeline(**common_kwargs)
    else:
        raise ValueError(f"Unknown pipeline type: {pipeline_type}")

# Creating the SDXL pipeline
sdxl_t2i_pipe = create_pipeline('text-to-image', model_name='sdxl')

**Comment by SoonNOON** [comment]
Sorry, I put this by mistake. I was testing if GPT4o understood how to implement something new.

**Comment by BasimBashir** [comment]
LLM's dont work like this besides we need any modern SDXL or FLUX unet trained to do relighting effect. As author is replacing SD1.5's unet with his custom trained one.
